package.json

```diff
@@ -620,19 +620,24 @@
         "lint": "",
         "$lint-fix": "tslint --project tsconfig.json -t verbose --fix",
         "package": "npm run webpack-prod && npm run get-language-server && gulp package",
-        "pretest": "npm run build && npm run webpack-prod",
-        "pretest-dev": "npm run build && npm run webpack",
-        "test": "gulp test",
-        "all": "npm i && npm test && npm    run lint",
-        "webpack": "gulp webpack-dev",
-        "webpack-prod": "gulp webpack-prod",
+        "pretest": "npm run pretest-prod",
```

Comment:
A bit of cleanup mostly, plus fixing this line:

"test": "node ./out/test/runTest.js",
=>
"test": "node ./dist/test/runTest.js",

Which ran the tests in two contexts causing weird and confusing errors.